parent/pom.xml

```diff
@@ -167,16 +167,46 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.4.1</version>
+                    <version>3.6.0</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-gpg-plugin</artifactId>
                     <!-- Older versions have issues with the gpg passphrase -->
-                    <version>3.0.1</version>
+                    <version>3.1.0</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.cyclonedx</groupId>
+                    <artifactId>cyclonedx-maven-plugin</artifactId>
+                    <version>2.7.9</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.asciidoctor</groupId>
+                    <artifactId>asciidoctor-maven-plugin</artifactId>
+                    <version>2.2.4</version>
                 </plugin>
             </plugins>
         </pluginManagement>
+
+        <plugins>
+            <plugin>
```

Comment:
Not sure about the change. The plugin will run in every module using this as parent, and for example in OpenMQ I see this:
```
[INFO] --- cyclonedx-maven-plugin:2.7.9:makeAggregateBom (default) @ mq ---
[INFO] CycloneDX: Resolving Dependencies
[INFO] CycloneDX: Creating BOM version 1.4 with 1 component(s)
[INFO] CycloneDX: Writing and validating BOM (XML): .../mq/main/target/bom.xml
[INFO]            attaching as mq-6.5.0-SNAPSHOT-cyclonedx.xml
[INFO] CycloneDX: Writing and validating BOM (JSON): .../mq/main/target/bom.json
[WARNING] Unknown keyword additionalItems - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
[INFO]            attaching as mq-6.5.0-SNAPSHOT-cyclonedx.json
```

Reply:
Is this some EF requirement? I don't understand why every child project is going to need this.

Reply:
Not at the moment, but there will eventually be a requirement to produce SBOMs coming up in the near future. This work is to be upfront with that. That said, if it generates a lot of problems for the projects, we can add it in a profile that can be disabled for those projects experiencing problems until it is fixed.